src/rawResponse.js

```diff
@@ -83,7 +83,7 @@ root.defaults = {
 	    foldGutter: true,
 	    gutters: ["CodeMirror-linenumbers", "CodeMirror-foldgutter"]
 	},
-	limit: 100
+	limit: 1000
```

Comment:
If results are more than 1000? What about then? Should we add some aler message that not all results are rendered as the reported suggested? Should or can we handle any ammout of results? I think not, but then, is it doable to have a `load more results` option in case not all results are loaded?

Reply:
The maximum results per page are 1000. When the user switch to the next page the following results will be shown.